### PR TITLE
Fix few things after removing InfluxDb logging

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -147,12 +147,17 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
```
Comment (hristoterezov) [inline pom.xml]: Is this dependancy related to the changes? Do we need it?

Comment (bgrozev) [inline pom.xml]: Wild guess: could be that the ice4j version which ends up being used is the one libjitsi depends on. We should update libjitsi's pom.xml and see if that helps.

Comment (paweldomas) [inline pom.xml]: hmm for some reason it selects ice4j 1.0 instead of ice4j 1.1 even if I bump the version in libjitsi. There is also jitsi-netaddr which depends on ice4j, but it does not have any version specified.
